agent/wptdriver/wpt_test.cc

```diff
@@ -157,6 +157,8 @@ void WptTest::Reset(void) {
   _run_error.Empty();
   _test_error.Empty();
   _custom_metrics.Empty();
+  _has_test_timed_out = false;

```

Comment:
Timeout concept only exists on a per-step basis, not a per-test basis. Hence a bool value needed to check if the whole test has timed out(rather than a step)